src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1332,22 +1354,33 @@
     <Resource Include="UI\Images\profile_normal.png" />
   </ItemGroup>
   <ItemGroup>
+    <Resource Include="UI\Images\pin_default_16_16.png" />
+    <Resource Include="UI\Images\watch_default_16_16.png" />
+    <Resource Include="UI\Images\wire_anchor_12_12.png" />
+    <Resource Include="UI\Images\watch_hidden_16_16.png" />
     <Resource Include="UI\Images\alert.png" />
-    <Resource Include="UI\Images\Pin_NoBackground_16px.png" />
+    <Resource Include="UI\Images\Pin_NoBackground_16_16.png" />
     <Resource Include="UI\Images\menu_16_16.png" />
     <Resource Include="UI\Images\caret_down.png" />
     <Resource Include="UI\Images\caret_up.png" />
     <Resource Include="UI\Images\PackageManager\close_16_16.png" />
     <Resource Include="UI\Images\PackageManager\down_16_16.png" />
     <Resource Include="UI\Images\PackageManager\folder_16_16.png" />
     <Resource Include="UI\Images\PackageManager\up_16_16.png" />
+    <Resource Include="UI\Images\pin_hidden_16_16.png" />
+    <Resource Include="UI\Images\pin_selected_16_16.png" />
+    <Resource Include="UI\Images\Pin_16_16.png" />
+    <Resource Include="UI\Images\Watch_16_16.png" />
     <EmbeddedResource Include="Controls\Docs\NodeAutocompleteDocumentation.html" />
     <Content Include="sharpdx_direct3d11_effects_x64.dll">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
     <Content Include="sharpdx_direct3d11_effects_x86.dll">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
+    <Resource Include="UI\Images\add_icon.png">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Resource>
```

Comment:
I think useually these resources are not copied, is there a reason this one is?

Reply:
same question

Reply:
Done.